.github/workflows/performance-test.yml

```diff
@@ -0,0 +1,76 @@
+name: Performance test
+
+on:
+  push:
+    branches:
+      - dev-2.x
+  pull_request:
+    branches:
+      - dev-2.x
+
+jobs:
+  perf-test:
+    if: github.repository_owner == 'opentripplanner'
+    runs-on: performance-test
+    steps:
+      - uses: actions/checkout@v2.3.2
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Set up Maven
+        uses: stCarolas/setup-maven@v4.2
+        with:
+          maven-version: 3.8.2
+
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Cache graph input files
+        uses: actions/cache@v2
+        with:
+          path: |
+            graph/*.osm.pbf
+            graph/*.zip
+          key: graph
+
+      - name: Build jar
+        run: mvn -DskipTests --batch-mode package
+
+      - name: Download OSM & Netex data
+        run: |
+          mkdir -p graph
+          wget https://download.geofabrik.de/europe/norway-210101.osm.pbf -O graph/norway.osm.pbf --no-clobber -q --show-progress --progress=bar:force || true
+          wget https://leonard.io/otp/rb_norway-aggregated-netex-2021-12-11.zip -O graph/rb_norway-aggregated-netex.zip --no-clobber -q --show-progress --progress=bar:force || true
+
+      - name: Build Norway graph
+        run: |
+          # copying the Entur build config but removing the "storage" section because that needs credentials
+          jq '. | del(.storage)' docs/examples/entur/build-config.json > graph/build-config.json
+          cp target/otp-2.1.0-SNAPSHOT-shaded.jar otp.jar
+          java -Xmx32G -jar otp.jar --build --save graph
+
+      - name: Copy graph, run speed test
+        env:
+          PERFORMANCE_INFLUX_DB_PASSWORD: ${{ secrets.PERFORMANCE_INFLUX_DB_PASSWORD }}
+          MEASUREMENT_ENVIRONMENT: CI
+        run: |
+          mv graph/graph.obj src/test/resources/raptor/speedtest/norway
+          mvn exec:java -Dexec.mainClass="org.opentripplanner.transit.raptor.speed_test.SpeedTest" -Dexec.classpathScope=test -Dexec.args="--dir=src/test/resources/raptor/speedtest/norway -p md -n 4"
```

Comment:
I think we should not mix the example with the performance test, so creating a separate directory with the needed files is my suggestion:
```
/test/ci_speed_test
  /travelSearch.csv
  /travelSearch-expected-results.csv
  /speed-test-config.json
```

Reply:
Then you do not have to modify the _build-config.json_ as well.

Reply:
Done